apps/website/app.vue

```diff
@@ -4,7 +4,9 @@
       <NuxtLayout>
         <NuxtPage />
       </NuxtLayout>
-      <GlobalToast />
+      <ClientOnly>
+        <GlobalToast />
```

Comment:
Hi @IonutzHerbil ! 
Can you please try renaming the component to UiGlobalToast so that we dont need the explicit import on line 17 in this component?

I it works, please remove the import on line 17 as well, thanks!